modules/bitgo/test/v2/unit/unspents.ts

```diff
@@ -0,0 +1,148 @@
+import * as nock from 'nock';
+import * as sinon from 'sinon';
+import { common, Wallet } from '@bitgo/sdk-core';
+import { TestBitGo } from '@bitgo/sdk-test';
+import { BitGo } from '../../../src';
+
+describe('Verify string type is used for value of unspent', function () {
+  const bitgo = TestBitGo.decorate(BitGo, { env: 'test' });
+  bitgo.initializeTestVars();
+  const basecoin: any = bitgo.coin('tdoge');
+  const walletData = {
+    id: '5b34252f1bf349930e34020a00000000',
+    coin: 'tdoge',
+    keys: [
+      '5b3424f91bf349930e34017500000000',
+      '5b3424f91bf349930e34017600000000',
+      '5b3424f91bf349930e34017700000000',
+    ],
+    coinSpecific: {},
+  };
+  const wallet = new Wallet(bitgo, basecoin, walletData);
+  const bgUrl = common.Environments[bitgo.getEnv()].uri;
+  const minValueStringMatcher = sinon.match('123');
+  const maxValueStringMatcher = sinon.match.string.and(sinon.match('10000000000000123'));
```

Comment:
it is unclear how and why you picked that number

this is clearer:
```
// cannot be represented by a JS Number
const highPrecisionBigInt = BigInt(1e16) + 1n;
```

Reply:
I'm not quite sure what the purpose of these sinon matchers is. You are using them with `test()` as a boolean func in one place and using them in assertion style in another place. 

Instead you can just write a method like

```
function matchMinMaxValue({ minValue: string, maxValue: string }): boolean {
  ...
}
```